x-pack/plugins/rollup/server/collectors/helpers.ts

```diff
@@ -168,7 +169,7 @@ export async function fetchRollupVisualizations(
     const visualizations = get(savedVisualizationsList, 'hits.hits', []);
     const sort =
       savedVisualizationsList.hits.hits[savedVisualizationsList.hits.hits.length - 1].sort;
-    visualizations.forEach(async (visualization: any) => {
```

Comment:
If have no idea why this is async, the body of this function seems synchronous to me. @spalger seems like this is an instance of 
> No need for async: There were a handful of places where the function passed to .forEach() was async unnecessarily

right?